### PR TITLE
docs: replace all apisix-openresty to apisix-base

### LINKED ISSUE #6983: docs: rename apisix-openresty to apisix-base
### Current State

https://github.com/apache/apisix/search?q=apisix-openresty

openresty is a separate trademark and does not belong to the Apache Software Foundation, so apisix-base should be used to make it clear to developers that this distribution has nothing to do with openresty

### Desired State

replace all apisix-openresty to apisix-base

### PR DESCRIPTION
**Pull request by oil-oil**
### Description

Fixes #6983

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by tokers** [comment]
@oil-oil There are several conflicts need to be fixed.

**Comment by oil-oil** [comment]
> @oil-oil There are several conflicts need to be fixed.

Okay, I forgot to sync the upstream master branch when I updated 🤓 .

File: docs/en/latest/admin-api.md [hunk 1]
```diff
@@ -564,7 +564,7 @@ The following should be considered when setting the `hash_on` value:
 - When set to `vars_combinations`, the `key` is required. The value of the key can be a combination of any of the [Nginx variables](http://nginx.org/en/docs/varindex.html) like `$request_uri$remote_addr`.
 - When no value is set for either `hash_on` or `key`, the key defaults to `remote_addr`.
 
-The features described below requires APISIX to be run on [APISIX-OpenResty](./how-to-build.md#step-6-build-openresty-for-apache-apisix):
+The features described below requires APISIX to be run on [APISIX-Base](./how-to-build.md#step-6-build-openresty-for-apache-apisix):
```
Comment (hf400159) [inline docs/en/latest/admin-api.md:567]: There is no `how-to-build.md#step-6-build-openresty-for-apache-apisix` in the latest how-to-build document, we need to change it to another link. We can add step 6 to the FAQ document. Or do we have any other option?

**Review by moonming — APPROVED** [review]

**Comment by moonming** [comment]
That will be great if we can add GitHub repo link for apisix-base

File: docs/en/latest/admin-api.md (hunk 1 above)
Comment (juzhiyuan) [reply docs/en/latest/admin-api.md:567]: Yes, @oil-oil please check https://apisix.apache.org/docs/apisix/next/how-to-build

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**